modules/tide_jira/src/Form/TideJiraConfigForm.php

```diff
@@ -110,6 +110,12 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Editor Department Field ID'),
     ];
 
+    $form['field_mappings']['fallback_department'] = [
+      '#type' => 'textfield',
+      '#default_value' => $config->get('fallback_department'),
+      '#title' => $this->t('Fallback department'),
```

Comment:
Could this do with a `#description` as well?

Reply:
hey @anthony-malkoun, thanks for the CR. yes, will add `#description`